tests/models/test_dagbag.py

```diff
@@ -937,8 +953,7 @@ def test_get_dag_with_dag_serialization(self):
     def test_collect_dags_from_db(self):
         """DAGs are collected from Database"""
         db.clear_db_dags()
-        example_dags_folder = airflow.example_dags.__path__[0]
-        dagbag = DagBag(example_dags_folder)
+        dagbag = DagBag(str(example_dags_folder))
```

Comment:
Maybe we should just allow `DagBag` to take `Path`, but this doesn’t need to be done now.

Reply:
Yeah, that was my thought -- it was orthogonal to this PR